back-end/api/quickstart/test_liked.py

```diff
@@ -0,0 +1,32 @@
+import json
+from rest_framework import status
+from django.test import TestCase, Client
+from .models import Like
+from .serializers import LikeSerializer
+from .helper_test import get_test_liked_fields
+
+client = Client()
+
+class GetLikedForAuthor(TestCase):
+  """Tests for getting all likes of a post at endpoint 'api/author/<str:author>/liked/'."""
```

Comment:
Tests for getting all things liked by an author.

Also, the endpoint is missing a / at the start, should be `/api/`